src/keys.js

```diff
@@ -23,6 +22,6 @@ export function publicKeyToAddress(publicKey: string) {
 }
 
 export function getPublicKeyFromPrivate(privateKey: string) {
-  const keyPair = new ECPair(bigi.fromHex(privateKey))
-  return keyPair.getPublicKeyBuffer().toString('hex')
+  const keyPair = ECPair.fromPrivateKey(new Buffer(privateKey, 'hex'))
```

Comment:
constructor is not exposed anymore need to use `ECPair.from***` methods. Also `fromPrivateKey` takes a regular `Buffer` not a `bigi.fromHex` result.